### DISCUSSION TITLE
🍄XSOSI 计划 2 月份数据

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#254] [2️⃣ 聊项目]
2022 年的 XSOSI 总共运行了 7 期：
- [6 月份数据](https://github.com/orgs/X-lab2017/discussions/115)
- [7 月份数据](https://github.com/X-lab2017/open-wonderland/discussions/149)
- [8 月份数据](https://github.com/X-lab2017/open-wonderland/discussions/173)
- [9 月份数据](https://github.com/X-lab2017/open-wonderland/discussions/196)
- [10 月份数据](https://github.com/X-lab2017/open-wonderland/discussions/210)
- [11 月份数据](https://github.com/X-lab2017/open-wonderland/discussions/221)
- [12 月份数据](https://github.com/X-lab2017/open-wonderland/discussions/231)

2023 年的 XSOSI 总共运行了 1 期：
- [1 月份数据](https://github.com/X-lab2017/open-wonderland/discussions/236)

2023 年的第 2 期，假期结束，涨幅明显（60%，达到新高）：
![image](https://user-images.githubusercontent.com/15010826/222166140-95aff06e-5739-4454-aca4-2fb805c15671.png)

详细情况参见：[这里](https://github.com/X-lab2017/open-digger/blob/master/notebook/community_analysis/xlab.ipynb)

XSOSI 大屏已同步更新（感谢 @zhicheng-ning ）：http://dataease.nzcer.cn/link/E1OQKVBz

![image](https://user-images.githubusercontent.com/15010826/222166498-7124b96f-0111-46aa-a97d-9a9fb28099b4.png)

**Comment by Nliver** [comment]
Cool~
